packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -15,6 +15,23 @@ function getTaskStatusDisplay(assignedTask?: string, taskStatus?: string) {
   }
 }
 
+function calculateLastUpdateTime(
+  updateTime: number,
+  lastUpdateTime: number,
+  timeOfLastUpdate: number,
+  setLastUpdateTime: any,
+  setTimeOfLastUpdate: any,
+): string {
+  return ((Date.now() - updateTime) / 1000).toFixed(2).toString();
```

Comment:
Tested with real-time on deployment demo setup (not with `use_sim_time`). The calculated last update time is accurate